sample-configs/noderesourcetopology.yaml

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
Does this sample make sense ? `NodeResourceTopology` is semantically not really a configuration custom resource. Instead it reflects the current state of the node.

Reply:
I didn't want to add it myself too but unfortunately I had to to pass the operator-sdk validation warning.
I understand that we don't expect users to interact with this CRD but olm expects you to place the CR instance of the CRD that the operator is owning.

Reply:
Hmm... I wonder if this CRD will get removed when we remove the operator. Also wonder what happens, if this CRD is already present when we install the operator.